### ISSUE TITLE
your code is not correct

### ISSUE THREAD

**Issue by qd213618** [leaderj1001/MobileNetV3-Pytorch#2]
there must be some mistakes in your code, because when i use your code as backbone to train my model the loss is not decreasing but maintaining. And when i change to https://github.com/kuan-wang/pytorch-mobilenet-v3 it do decreased again! so, check your code, please.

**Comment by leaderj1001** [comment]
Thank you. I'll check it once.
